include/node.h

```diff
@@ -0,0 +1,88 @@
+#ifndef PIPE_NETWORK_NODE_H_
+#define PIPE_NETWORK_NODE_H_
+
+#include <Eigen/Dense>
+
+//! Node class
+//! \brief Class that stores the information about nodes
+class Node {
+
+ public:
+  // Constructor with id and coordinates
+  //! \param[in] id node id
+  //! \param[in] coordinates coordinates of the node
+  Node(unsigned id, const Eigen::Vector3d& coordinates)
+      : id_{id}, coordinates_{coordinates} {}
+
+  //! Destructor
+  ~Node() = default;
+
+  //! Copy constructor
+  Node(const Node&) = delete;
+
+  //! Assignment operator
+  Node& operator=(const Node&) = delete;
+
+  //! Move constructor
+  Node(Node&&) = delete;
+
+  //! Return id
+  //! \retval id_ return id of the node
+  unsigned id() const { return id_; }
+
+  //! Return coordinates
+  //! \retval coordinates_ return coordinates of the node
```

Comment:
\reval means return, just explain what it is instead of `return coordinates of the node`, `coordinates_ coordinates of the node`